### DISCUSSION TITLE
Inline edit doesn't udpate data.

### DISCUSSION THREAD

**Discussion by mferrerAPI** [dbeaver/dbeaver#19467] [Q&A]
Hello, 
I'm updating values from a select, I press the Save button, I see this screenshot I select Update and seems everything is fine.

![image](https://user-images.githubusercontent.com/17877814/227262231-27923803-1d5d-4dd2-9b5d-7d25bc9a52e8.png)

But if I do the select again, old data still appearing. Data is not being updated.

I am using DBeaver community edition Version 23.0.0.202303040621
and connecting to snowflake DDBB

What I am missing? THank you.

**Comment by mkgrgis** [comment]
There will be no ideas, @mferrerAPI without DBeaver version and name of database management system type. Oracle?

**Reply by mferrerAPI** [reply]
Thanks, I've updated the main thread.

I am using DBeaver community edition Version 23.0.0.202303040621
and connecting to snowflake DDBB

**Comment by LonwoLonwo** [comment]
It can be problem with the date representation. You need to check that time is correct and not changed because of the Time Zone. 

You can try to disable date/time formatting also.

![2023-03-24 11_13_04-Window](https://user-images.githubusercontent.com/45152336/227462604-00f3436d-6ccf-4c70-a4cd-4fa5d1abd360.png)

**Reply by mferrerAPI** [reply]
That's it!!
Thank you 😃
